core/trino-main/src/main/java/io/trino/server/TrinoSystemRequirements.java

```diff
@@ -60,6 +62,13 @@ private static void verifyJvmRequirements()
         verifyUtf8();
     }
 
+    private static void verifyManagementBeans()
+    {
+        if (!(ManagementFactory.getOperatingSystemMXBean() instanceof OperatingSystemMXBean)) {
+            failRequirement("Trino requires access to the com.sun.management MBeans");
```

Comment:
If this access was not there earlier, was it failing startup or queries, or just missing JMX stats ?

Reply:
It would actually return empty CPU load statistics (which we don't want).